scripts/deploy-to-dist.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Pull requests and commits to other branches shouldn't try to deploy
+if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$TRAVIS_BRANCH" != "public-api" ]; then
+  echo "Skip deploy: $TRAVIS_BRANCH"
+  exit 0
+fi
+
+echo "Starting deployment"
+echo "Target: dist branch"
+
+DIST_DIR="dist"
+TMP_DIST_DIR="tmp_dist"
+PUBLIC_DIR="public"
+INTERNAL_DIR="internal"
```

Comment:
There is no need to "upload" the internal dist files to this branch IMO.

Reply:
This can be useful for the Builder in the future

Reply:
But I can remove it for now